docs/tutorials/material_solvated.ipynb

```diff
@@ -170,10 +175,12 @@
    "source": [
     "si_layer = Layer.from_pars(si, 0, 0, 'Si layer')\n",
     "sio2_layer = Layer.from_pars(sio2, 30, 3, 'SiO2 layer')\n",
+    "subphase = Multilayer.from_pars([si_layer, sio2_layer], name='Si/SiO2 Subphase')\n",
+    "\n",
     "solvated_film_layer = Layer.from_pars(solvated_film, 250, 3, 'Film Layer')\n",
-    "superphase = Layer.from_pars(d2o, 0, 3, 'D2O Subphase')\n",
     "\n",
-    "subphase = Multilayer.from_pars([si_layer, sio2_layer], name='Si/SiO2 Superphase')\n",
+    "superphase = Layer.from_pars(d2o, 0, 3, 'D2O Superphase')\n",
+    "\n",
```

Comment:
I'm not 100% sure, but is the neutron "entering" the sample from the first entry of  `Sample.from_pars()`? If so, then this is not a realistic experimental geometry. The neutrons will always enter from the silicon substrate, or air, not the liquid. If not then the D2O and silicon/SiO2 are mislabelled, since the superphase should be the material from which the neutrons are incident, and the subphase is the material which they can be partially transmitted into.

Reply:
Hmm, good question.  We will need to take a careful look at this and I have parked the potential problem in a new issue for now.  The Notebook for simple_fitting.ipynb seems to seems to have a similar issue.

https://github.com/easyScience/EasyReflectometryLib/issues/116

Reply:
@arm61 are you able to answer this?